Fix java dependency with apache client (#1756)
ClickHouse/clickhouse-docs: docs/en/integrations/language-clients/java/index.md

</dependency>
```

> Since version `0.5.0` the driver uses a new client http library that need to added as dependency.

Reviewer: ```suggestion
Since version `0.5.0`, the driver uses a new client http library that needs to be added as a dependency.
```

    <classifier>all</classifier>
</dependency>
```

> Since version `0.5.0` we are using Apache HTTP Client we are packing the Client, but since there is not a sharded version we need to have a logger as dependency (if not already using please include as dependency).

Reviewer: ```suggestion
Since version `0.5.0`, we are using Apache HTTP Client that's packed the Client. Since there is not a shared version of the package, you need to add a logger as a dependency.
```
